client/my-sites/plans/main.jsx

```diff
@@ -69,105 +61,16 @@ import WooExpressPlansPage from './woo-express-plans-page';
 
 import './style.scss';
 
-function DomainAndPlanUpsellNotice() {
-	const translate = useTranslate();
-	const noticeTitle = translate( 'Almost done' );
-	const noticeDescription = translate( 'Upgrade today to claim your free domain name!' );
-	return (
-		<Banner
-			title={ noticeTitle }
-			description={ noticeDescription }
-			icon="star"
-			showIcon
-			disableHref
-		/>
-	);
-}
-function DescriptionMessage( { isDomainUpsell, isFreePlan, yourDomainName, siteSlug } ) {
-	const translate = useTranslate();
-	const dispatch = useDispatch();
-
-	if ( ! isDomainUpsell ) {
-		return (
-			<>
-				<p>
-					{ translate(
-						'With your annual plan, you’ll get %(domainName)s {{strong}}free for the first year{{/strong}}.',
-						{
-							args: {
-								domainName: yourDomainName,
-							},
-							components: {
-								strong: <strong />,
-							},
-						}
-					) }
```

Comment:
I see we replaced it with [this upsell](https://github.com/Automattic/wp-calypso/blob/trunk/client/landing/stepper/declarative-flow/internals/steps-repository/plans/plans-wrapper.tsx#L222-L232) which is nice so we're reducing complexity on this page.